engines/sci/engine/script_patches.cpp

```diff
@@ -7146,22 +7148,22 @@ static const uint16 qfg1vgaSignatureWhiteStagDagger[] = {
 	0x1a,                               // eq?
 	0x31, 0x16,                         // bnt [next parameter check]
 	0x76,                               // push0
-	0x45, 0x02, 0x00,                   // callb export 2 from script 0, 0
+	0x45, 0x02, 0x00,                   // callb 0 proc0_2
 	SIG_MAGICDWORD,
-	0x38, SIG_SELECTOR16(say),          // pushi 0127h (selector "say")
+	0x38, SIG_SELECTOR16(say),          // pushi say
 	0x39, 0x05,                         // pushi 05
 	0x39, 0x03,                         // pushi 03
 	0x39, 0x51,                         // pushi 51h
 	0x76,                               // push0
 	0x76,                               // push0
 	0x7c,                               // pushSelf
-	0x81, 0x5b,                         // lag global[5Bh] -> qg1Messager
+	0x81, 0x5b,                         // lag global[91] -> qg1Messager
 	0x4a, 0x0e,                         // send 0Eh -> qg1Messager::say(3, 51h, 0, 0, stagHurt)
-	0x33, 0x12,                         // jmp -> [ret]
+	0x33, 0x12,                         // jmp [end the switch]
```

Comment:
should be "end of routine" / "end of method". The original comment was more obvious where the jump is supposed to lead to.